src/Aspire.Hosting.Azure.CognitiveServices/Aspire.Hosting.Azure.CognitiveServices.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <IsPackable>true</IsPackable>
-    <PackageTags>aspire hosting azure</PackageTags>
-    <Description>Azure Open AI resource types for .NET Aspire.</Description>
+    <PackageTags>aspire hosting azure openai ai</PackageTags>
```

Comment:
https://www.nuget.org/packages/Azure.ResourceManager.CognitiveServices/ also has the `cognitiveservices` tag. Should we add it here as well?